packages/hoprd/src/index.ts

```diff
@@ -27,7 +27,7 @@ const argv = yargs
     default: '0.0.0.0:9091'
   })
   .option('announce', {
-    boolean: false,
```

Comment:
not sure whether this should default to `true`

Reply:
hmm not sure, then the command should be `--not-announce`, let's wait on @peterbraden input for this

Reply:
This needs to default to false. Normal clients should not add routing information.